app/extensions/intelligent_agent/models.py

```diff
@@ -536,7 +535,9 @@ def detection_complete(self):
             if self.owner and not self.owner.is_public_user():
                 ags = self.get_asset_group_sighting()
                 assert ags, f'no AGS for {self}'
-                url = full_api_url(f'pending-sightings/{str(ags.guid)}')
+                url = url_for(
+                    'frontend.pending-sightings', guid=str(ags.guid), _external=True
+                )
```

Comment:
yay thanks for going in and taking care of my use-cases!

Reply:
Oh yeah, could you check that it's correct?  There's no tests so I'm not sure